GTBitsOfGood/canine-assistants
# Sign up and log in pages [styling & design]

## Styled skeleton sign up and log in pages
Issue Number(s): #23 

What does this PR change and why?
### Checklist

### How To Test
Execute the command ```npm run dev``` to start the development server.
Go to localhost:<PORT>/login or localhost:<PORT>/signup to view the log in and sign up pages respectively.
Use inspect to ensure that the styling matches the designs in the [CA Figma](https://www.figma.com/file/TQGCR28rm4H8OG8cQwDYSF/Canine-Assistants-FALL23?type=design&node-id=1-2&mode=design).
Additional testing: resize the window to ensure the webpage doesn't break & use a screen reader to ensure the site is accessbile.
